src/app/component/eco-news/components/create-news/create-news.component.ts

```diff
@@ -30,8 +30,9 @@ export class CreateNewsComponent implements OnInit {
     {name: 'Ads', isActive: false}
   ];
 
-  public activeLanguage = 'en';
-  private date: Date = new Date();
+  private year: number = new Date().getFullYear();
+  private day: number = new Date().getDate();
+  private month: number = new Date().getMonth();
```

Comment:
Actually, you are using the variable <code>year</code> and other in the template, e.g. here:
```html
 <div class="date">
                <p><span>{{ 'create-news.date' | translate }}</span>
                  <span>{{day}} {{'date.months.' + [month] | translate }} {{year}} {{'date.year' | translate }}</span></p>
                <p><span>{{ 'create-news.author' | translate }}</span><span>{{ 'create-news.author-name' | translate }}</span></p>
            </div>
```
So, they shouldn't be <code>private</code>